pkg/distributor/distributor.go

```diff
@@ -412,6 +413,10 @@ func New(cfg Config, clientConfig ingester_client.Config, limits *validation.Ove
 			Name: "cortex_distributor_latest_seen_sample_timestamp_seconds",
 			Help: "Unix timestamp of latest received sample per user.",
 		}, []string{"user"}),
+		labelValuesWithNewlinesPerUser: promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
+			Name: "cortex_distributor_label_values_with_newlines_total",
+			Help: "Total amount of label values with newlines seen at ingestion time.",
```

Comment:
nit: amount -> number. "amount" is used for uncountable items.

Reply:
Done in https://github.com/grafana/mimir/pull/9400/commits/faa9ea4b6bdb888d600fb80d195885fe27d332e3